proto/allocation/v1/tx.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package allocation.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/any.proto";
+import "allocation/v1/allocation.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/allocation/types";
+
+// MsgService defines the msgs that the oracle module handles.
+service Msg {
+     // DelegateAllocations defines a message that delegates the allocating to an account address.
+    rpc DelegateAllocations (MsgDelegateAllocations) returns (MsgDelegateAllocationsResponse);
+    // OracleDataPrevote defines a message that commits a hash of a oracle data feed before the data is actually submitted.
+    rpc AllocationPrecommit   (MsgAllocationPrecommit)   returns (MsgAllocationPrecommitResponse);
+    // OracleDataVote defines a message to submit the actual oracle data that was committed by the feeder through the prevote.
+    rpc AllocationCommit      (MsgAllocationCommit)      returns (MsgAllocationCommitResponse);
+}
```

Comment:
Do we still want precommit/reveal here if we are doing sparse voting?

Reply:
Never heard the term "sparse voting" so if we aren't, it is news to me